bin/p2-dsctl/main.go

```diff
@@ -22,18 +22,19 @@ import (
 )
 
 const (
-	CmdCreate  = "create"
-	CmdGet     = "get"
-	CmdList    = "list"
-	CmdEnable  = "enable"
-	CmdDisable = "disable"
-	CmdDelete  = "delete"
-	CmdUpdate  = "update"
+	CmdCreate           = "create"
+	CmdGet              = "get"
+	CmdList             = "list"
+	CmdEnable           = "enable"
+	CmdDisable          = "disable"
+	CmdDelete           = "delete"
+	CmdUpdate           = "update"
+	CmdOverrideSelector = "overrideSelector"
 )
 
 var (
 	cmdCreate       = kingpin.Command(CmdCreate, "Create a daemon set.")
-	createAZ        = cmdCreate.Flag("az", "The availability zone of the pod cluster").Required().String()
+	createAZ        = cmdCreate.Flag("az", "The availability zone of the pod cluster").String()
```

Comment:
What does this AZ change have to do with: "Add a an option to override the ds node selector"?